include/dos_system.h

```diff
@@ -419,7 +419,7 @@ void VFILE_Register(const char *name,
                     const char *dir = "");
 void VFILE_Register(const char* name, const std::vector<uint8_t>& blob,
                     const char* dir = "");
-void VFILE_Update(const char* name, std::vector<uint8_t> blob, const char* dir = "");
+bool VFILE_Update(const char* name, const std::vector<uint8_t> &blob, const char* dir = "");
```

Comment:
Can you rename `blob` to something actually meaningful while you're at it? Please 😅 

All these names carry the exact same amount of information (zero):

- blob
- data
- theThing
- asdf
- bytes
- ...